test/acceptance/tests/hcp/hcp_test.go

```diff
@@ -1,124 +1,119 @@
 package hcp
 
 import (
-	"encoding/json"
+	"context"
 	"fmt"
 	"strings"
 	"testing"
 	"time"
 
 	terratestLogger "github.com/gruntwork-io/terratest/modules/logger"
 	"github.com/gruntwork-io/terratest/modules/random"
-	"github.com/gruntwork-io/terratest/modules/shell"
 	"github.com/gruntwork-io/terratest/modules/terraform"
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/hashicorp/terraform-aws-consul-ecs/test/acceptance/framework/config"
 	"github.com/hashicorp/terraform-aws-consul-ecs/test/acceptance/framework/helpers"
 	"github.com/hashicorp/terraform-aws-consul-ecs/test/acceptance/framework/logger"
 	"github.com/stretchr/testify/require"
 )
 
+const setupDir = "../../setup-terraform"
+
+var timeouts = map[string]*retry.Timer{
```

Comment:
It might be cleaner to define and use a struct since we know all of the keys. I might be missing something though.

Reply:
Great suggestion!

Reply:
Yeah, that is even better @pglass